### ISSUE TITLE
Save port settings in profile

### ISSUE THREAD

**Issue by pizzapim** [nstlaurent/DoomLauncher#215]
I am playing a GZDoom mod simultaneously with another WAD and need different settings in GZDoom for bothof them. It would be useful to save the settings of the port (key bindings, comp level, etc.) in the profile, so I don't have to change the settings everytime I switch.

**Comment by nstlaurent** [comment]
You can make multiple config files in GZDoom and force load with the -config parameter. Make your alternate config file and in the Extra Params section in Doom Launcher add:
-config alternateconfig.ini

https://zdoom.org/wiki/Command_line_parameters

**Comment by pizzapim** [comment]
Thanks! Works like a charm!

**Comment by MasterZeroFlash** [comment]
> You can make multiple config files in GZDoom and force load with the -config parameter. Make your alternate config file and in the Extra Params section in Doom Launcher add: -config alternateconfig.ini
> 
> https://zdoom.org/wiki/Command_line_parameters

Recently started using a launcher called DoomRunner (which is great), but was lured in by the auto mod database function of DoomLauncher and there is just a tiny thing, with DoomRunner (which in many ways works the same, both are launchers after all) in the main window for a "profile" (whatever one decides to name n number of mods or maps etc put together specifically (X + Y mod as opposed to Z + Y mod etc), you have a dropdown menu for selecting what ini config to use (from the ones available at your chosen sourceport directory/ies) and you can even just choose the default one (GZDoom.ini or whatever it was for me) and then make a copy of that ini in-program with just one press of a button, and Im wondering if there is something similar for DoomLauncher or if the quoted manual way is still the way to do it? (I mean DoomRunner just takes care of adding the command prompts so nothing too special, but considering I play around with many different mods and map packs I more often than not make a new ini for each file, and I feel that will be prone to screwups >_<. 

...how did that take so many words to say..

TLDR: Can the managing of ini files and adding the appropriate -config parameter to the launch parameters of a game profile be made more automatic (like being able to choose ini for a profile and having said ini being automatically added as a -config param, or being able to manage ini files from within the program, like on can with DoomRunner)? (Basically the whole Profile thing and being able to copy and whatnot, but with INI files).

Sorry if Im being a dumb, I blame influenza :v

EDIT: Hmmm, Im not sure DoomLauncher does the thing Im looking for though, or maybe I just need to learn the program, oh well. (The database connection is a pretty brilliant idea though).
